app/src/main/res/layout/episodes_list_fragment.xml

```diff
@@ -28,7 +28,7 @@
             android:text="@string/filtered_label"
             android:layout_marginTop="-12dp"
             android:layout_marginLeft="60dp"
-            android:layout_marginStart="60dp"
+            android:layout_marginStart="15dp"
```

Comment:
This changes MarginStart but keeps MarginLeft. I would just add MarginHorizontal and set it to 16dp

Reply:
What if we remove MarginLeft, since MarginStart does the job for us.

Reply:
If the text gets long, it might be useful to have symmetric margins. 15dp is a very uncommon number on Android. It uses multiples of 8.

Reply:
This is why I suggested marginHorizontal. That's just one line and more descriptive

Reply:
Ok then I will create a new PR with this solution.

Reply:
> Ok then I will create a new PR with this solution.

You can just push more commits, no need to create a new PR

Reply:
Thinking about it some more, I think using padding instead of margin would be better because then the background highlight effect when clicking the items fills the whole width

Reply:
Yes i just tried the padding and the background highlight effect is also coming out in a better way